components/navigation/Filter.js

```diff
@@ -0,0 +1,40 @@
+import {useState} from 'react';
+import PropTypes from 'prop-types';
+
+export default function Filter({data, onFilter}) {
+    const [state, setState] = useState('Filter');
+    const catgeories = [];
+    data.map((data) => {
+        catgeories.push(data.category)
+    });
+    const onFilterApply = (value) => {
+        setState(value);
+        const jobs = data.filter((e) => e.category === value);
+        onFilter(jobs);
+    }
+    return (
+        <div class="mb-4">
+            <div className="select-box inline-flex w-56 max-w-sm justify-between">
```

Comment:
I just recalled we already have a Select component: https://github.com/asyncapi/website/blob/master/components/form/Select.js. Please use this one. If you're curious how it looks like, it's the one here: https://www.asyncapi.com/generator.

Reply:
Ohh... So I should get rid of the one here right? I'll replace it and update the PR

Reply:
But mine looks more fancy Lol 😂. I'll replace it then.

Reply:
Thanks for the review tho

Reply:
haha! It looks better indeed. Feel free to improve the existing one 😄 There's something that's also better in the existing one and it's that it uses a native `select` HTML tag instead of a div.